ansible_base/tests/functional/test_organization_model.py

```diff
@@ -0,0 +1,46 @@
+from datetime import datetime
```

Comment:
Can we move this under unit instead of functional? Since it's not using the django test client to call into the system.

Generally we've been separating the tests as: If it's just testing individual functions or behaviors, or creating things directly, then it goes under unit; if it's using the client to make actual (fake) http calls into the API, then it goes under functional.